ui/components/src/StatChart.tsx

```diff
@@ -56,6 +53,15 @@ const noDataOption = {
   series: [],
 };
 
+export type UnixTimeMs = number;
+
+export type GraphSeriesValueTuple = [timestamp: UnixTimeMs, value: number];
+
+export interface GraphSeries {
+  name: string;
+  values: Iterable<GraphSeriesValueTuple>;
+}
```

Comment:
This is temporary to fix circular dependency errors btw, planning to move these out of StatChart